kong/plugins/hmac-auth/access.lua

```diff
@@ -1,24 +1,20 @@
-local utils = require "kong.tools.utils"
-local responses = require "kong.tools.responses"
 local constants = require "kong.constants"
-local openssl_hmac = require "openssl.hmac"
-local resty_sha256 = require "resty.sha256"
-
-local math_abs = math.abs
-local ngx_time = ngx.time
-local ngx_gmatch = ngx.re.gmatch
-local ngx_decode_base64 = ngx.decode_base64
-local ngx_encode_base64 = ngx.encode_base64
-local ngx_parse_time = ngx.parse_http_time
-local ngx_set_header = ngx.req.set_header
-local ngx_get_headers = ngx.req.get_headers
-local ngx_log = ngx.log
-local req_read_body = ngx.req.read_body
-local req_get_body_data = ngx.req.get_body_data
-local ngx_hmac_sha1 = ngx.hmac_sha1
-local split = utils.split
```

Comment:
ditto - using `utils.split` is more consistent, because changing the underlying implementation of `split` can be done in a single place. Again, the goal isn't to get rid of `kong.tools.utils` here.

Reply:
Fixed already.